ddtrace/tracer.py

```diff
@@ -331,6 +333,18 @@ def configure(
         if (collect_metrics is None and runtime_metrics_was_running) or collect_metrics:
             self._start_runtime_worker()
 
+        info = debug.collect(self)
+        if self.log.isEnabledFor(logging.INFO) and environ.get("DD_TRACE_STARTUP_LOGS") != "0":
```

Comment:
Please use `asbool` to check `DD_TRACE_STARTUP_LOGS` value

Reply:
This is intentional as to be consistent with the other libraries although it breaks consistency within our library 😩

Reply:
Using `asbool` is just going to make things such as `false` work. I expect that people used to this library or not, would be happy to see that value being respected.
I think we can have consistency with our own tracer, the other traces, and user-friendlyness, at the same time. :)